src/python/pants/notes/1.28.x.rst

```diff
@@ -143,7 +143,7 @@ Bugfixes
 * Remove deprecation warning from ivy_imports.py. (#9696)
   `PR #9696 <https://github.com/pantsbuild/pants/pull/9696>`_
 
-* Fix `PYTHONPATH` comingling in sitepackages. (#9690)
+* Fix `PYTHONPATH` commingling in sitepackages. (#9690)
```

Comment:
Huh, this is weird, but indeed that's how you spell the word, it seems.

Reply:
Yeah, I had to search and read a dictionary on it.

Reply:
For people curious, here's the explanation from https://www.lexico.com/definition/commingle:
https://www.lexico.com/definition/com-
> Com- is used before b, m, p, also occasionally before vowels and f. The following variant forms occur: co- especially before vowels, h, and gn; col- before l; cor- before r; and con- before other consonants